tools/flagd-http-connector/checkstyle-suppressions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
both checkstyle files should not be here as part of the parent, but CI build is failing on checkstyle without them